lib/ModuleGraph.js

```diff
@@ -750,6 +750,25 @@ class ModuleGraph {
 		return mgm.async;
 	}
 
+	/**
+	 * @param {Module} module the module
+	 * @returns {boolean} true, if the module is used as a deferred module at least once
```

Comment:
This function actually returns `true` only if the module is _always_ used as a deferred module. Do we have a test where it's imported first as deferred and then as non-deferred?
```js
// entry
import "a"
import "b"
```
```js
// a
import defer "c"
```
```js
import "c"
```

Reply:
No, I tested it and found it's doing wrong, let me fix it

Reply:
fixed, I hope, it's a rabbit hole